modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/dsl/Monitoring.scala

```diff
@@ -69,6 +115,22 @@ object Monitoring {
       def reportMetrics(metrics: Metrics.KVMetrics): F[Unit] =
         reporters.traverse_(r => r.report(metrics.toList))
 
+      def alert(payload: AlertPayload): F[Unit] =
+        webhookConfig match {
+          case Some(webhook) =>
+            val request: Request[F] =
+              Request[F](Method.POST, webhook.endpoint)
+                .withEntity(payload.copy(tags = payload.tags ++ webhook.tags))
+
+            httpClient
+              .run(request)
+              .use { response =>
+                if (response.status.isSuccess) Sync[F].unit
+                else response.as[String].flatMap(body => Logging[F].error(s"Webhook ${webhook.endpoint} returned non-2xx response:\n$body"))
```

Comment:
The problem is that it's not even an exception. It's just a typical non-2xx response. If collector returns non-2xx response - operators would have other means to know about it. If it's something more like a real error, such as DB connection issue - it will be raised to Sentry downstream.